packages/lit-dev-tools-cjs/src/playground-plugin/plugin.ts

```diff
@@ -318,15 +318,20 @@ export const playgroundPlugin = (
       }
 
       const noHeaderAttribute = noHeader === 'no-header' ? ' no-header' : '';
+
+      // Matches the first line's leading whitespace and applies it to the other
+      // HTML nodes to preserve the same indentaiton.
+      const contentIndent = (content.match(/^\s*/) ?? [''])[0];
```

Comment:
Nit: Because `*` can match zero spaces, as far as I can tell, this will never fail to match and never return `null`. Any string I test without spaces will match zero spaces and return `''`.
Maybe can remove the `?? ['']`.

Reply:
Oh I see this makes the type checker happy. Resolving.